src/thorlabs_cube/driver/tcube/tpz.py

```diff
@@ -348,28 +350,28 @@ async def get_output_lut_parameters(self):
         get_msg = await self.send_request(
             MGMSG.PZ_REQ_OUTPUTLUTPARAMS,
             [MGMSG.PZ_GET_OUTPUTLUTPARAMS],
-            1,
+            _CHANNEL,
         )
         return st.unpack("<HHLLLL", get_msg.data[2:22])
 
-    async def start_lut_output(self):
+    async def start_lut_output(self) -> None:
         """Start the voltage waveform (LUT) outputs."""
-        await self.send(Message(MGMSG.PZ_START_LUTOUTPUT, param1=1))
+        await self.send(Message(MGMSG.PZ_START_LUTOUTPUT, param1=_CHANNEL))
 
-    async def stop_lut_output(self):
+    async def stop_lut_output(self) -> None:
         """Stop the voltage waveform (LUT) outputs."""
-        await self.send(Message(MGMSG.PZ_STOP_LUTOUTPUT, param1=1))
+        await self.send(Message(MGMSG.PZ_STOP_LUTOUTPUT, param1=_CHANNEL))
```

Comment:
```suggestion
        await self.send(Message(MGMSG.PZ_STOP_LUTOUTPUT, param1=Tpz._CHANNEL))
```
Use class constants.